src/Symfony/Component/Translation/Extractor/Visitor/AbstractVisitor.php

```diff
@@ -52,22 +52,21 @@ protected function getStringArguments(Node\Expr\CallLike|Node\Attribute|Node\Exp
             return [];
         }
 
-        /** @var Node\Arg $arg */
-        $arg = $args[$index];
-        if (!$result = $this->getStringValue($arg->value)) {
-            return [];
+        if (($arg = $args[$index]) instanceof Node\Arg) {
```

Comment:
```suggestion
        $args = $node instanceof Node\Expr\CallLike ? $node->getRawArgs() : $node->args;

        if (($arg = $args[$index]) instanceof Node\Arg) {
```
The `$args` variable has been moved from 45 line